src/cdk/v2/bindings/default.js

```diff
@@ -1,4 +1,4 @@
-const { TransformationError } = require("../../../v0/util");
+const { TransformationError } = require("../../../v0/util/errorTypes");
```

Comment:
should we use instrumentation error here?

Reply:
CC : @saikumarrs

Reply:
@koladilip yes. CC @mihir-4116